client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcVendorCompatibility.java

```diff
@@ -370,10 +370,9 @@ private void checkRepresentation(String trinoExpression, List<String> referenceD
                     checkRepresentation(trinoExpression, referenceExpression, type, sessionTimezoneId, driver, assertion);
                 }
                 catch (RuntimeException | AssertionError e) {
-                    String message = format("Failure when checking behavior against %s", driver);
```

Comment:
This used to be better as variable was reused, not it is less DRY

Reply:
Yes, this is less DRY, but I don't mind some duplication and I didn't want to introduce any behavioral changes.

> do we need to log when we throw?

I would guess not. Assuming that the error is logged somewhere else.

Reply:
> I didn't want to introduce any behavioral changes.


then change to `log.error(e, "%s", message)`

Reply:
Yes, that's what I figured as well. I was just hesitating between this and tracing if we actually do need to log before throwing. But this is simpler :)